stories/components/settings/medtronic/Medtronic.js

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 
 import { storiesOf } from '@kadira/storybook';
 
-import Medtronic from '../../../../src/components/settings/medtronic/Medtronic';
+import { getChart } from '../../../../src/utils/settings/factory';
+const Component = getChart('carelink');
```

Comment:
I didn't like calling it a "chart" because it's not a chart. But I can change it to `Settings` or `SettingsComponent`?